RCS file: /pack/cvsroots/wxwindows/wxWindows/src/generic/grid.cpp,v
retrieving revision 1.260
diff -r1.260 grid.cpp
3916,3917c3916,3924
<         if (m_ownTable) 
<             delete m_table; 
---
>         // stop all processing 
>         m_created = FALSE; 
> 
>         if (m_ownTable)
>         {
>             wxGridTableBase *t=m_table;
>             m_table=0;
>             delete t; 
>         }
3920d3926
<         // stop all processing 
3923d3928
<         m_created = FALSE; 
 	  	 
